tasks/natural-language-processing/sentiment-analysis/Pritam/deep_sentiment_analysis.py

```diff
@@ -0,0 +1,193 @@
+                print("text is same for optinion and fact OR not opinion or fact but compound")
+                check = check + "text is same for optinion and fact OR not opinion or fact but compound. "
+                print("positive compound")
+                check = check + "positive compound. "
+            elif blob.sentiment.polarity == 0:
+                print("text is same for optinion and fact OR not opinion or fact but compound")
+                check = check + "text is same for optinion and fact OR not opinion or fact but compound. "
+                print("neutral compound")
+                check = check + "neutral compound. "
+            else:
+                print("text is same for optinion and fact OR not opinion or fact but compound")
+                check = check + "text is same for optinion and fact OR not opinion or fact but compound. "
+                print("negative compound")
+                check = check + "negative compound. "
+    else:
+            print("text is not compound but opinion or fact") 
+            check = check + "text is not compound but opinion or fact. "
+    print("\n")
+    
+    return check
+            
+        
+
+
+
+
+def write_combined_string_data_to_text_file(check, input_text):
+    
+       
+    
+    data_1 = "Hi, Pritam. I am a program of NLP and computer vision. "
+    data_2 = "The input text file data is- . " + input_text
+    data_3 = "The input text ends. As a mention, By my intelligence, I can understand that the sentiment of your input is-. Your ." + check + "Hope to meet you again. Bye."
+
+    utterences = data_1 + data_2 + data_3 
+
+    print(utterences)
+
+    utterences_file_path = "utterences.txt" 
+
+    with open(utterences_file_path, "w") as write_utterences_to_file:
+        write_utterences_to_file.write(utterences)
+        write_utterences_to_file.close()
+        
+    return utterences, utterences_file_path
+    
+
+
+
+
+
+if __name__ == "__main__":
+
+    
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument("--text_data_file_input_path", type=str, default = "summary_file.txt", help='input a string')
+
+    args = parser.parse_args()
+
+    print(args.text_data_file_input_path)
+
+
+
+    input_file_path = args.text_data_file_input_path
+    input_data_file = open(input_file_path, "r")
+    input_text = input_data_file.read()
+    input_data_file.close()
+
+
+    blob = input_text_to_blob(input_text)
+ 
+
+    check = intelligent_sentiment_checker(blob)
+
+    print("-----------------------------------------")
+    
+    utterences, utterences_file_path = write_combined_string_data_to_text_file(check, input_text)
+
+    
+    print("-----------------------------------------")
+    
+    if not os.path.isdir("client_uri"):
+        os.makedirs("client_uri")
+        
+    mlflow.tracking.set_tracking_uri("client_uri")
+    mlflow.tracking.get_tracking_uri()
+    
+    
+    client = MlflowClient()
```

Comment:
Could you please update mlflow section in the code. it's giving error while running code second time onwards as experiment names are hard coded. Please refer this [MLflow example](https://github.com/mlflow/mlflow/blob/master/examples/sklearn_elasticnet_wine/train.py) and keep options required for logging as of now.
Or please do let me know if there is another way to run the task?